lib/chat_api_web/views/customer_view.ex

```diff
@@ -50,7 +50,8 @@ defmodule ChatApiWeb.CustomerView do
       current_url: customer.current_url,
       browser: customer.browser,
       os: customer.os,
-      metadata: customer.metadata
+      metadata: customer.metadata,
+      title: customer.name || customer.email || "Anonymous User"
```

Comment:
I noticed we had similar logic in different places, but would render different content. I figured this would be a good way to consolidate that so we don't have to duplicate that logic in too many places.

Reply:
seems reasonable!